lib/features/support/app-automate.rb

```diff
@@ -0,0 +1,121 @@
+require 'appium_lib'
```

Comment:
It might be preferable to rename this file to app_automate_driver to maintain consistency with the class that this file defines.